docs/support/account.md

```diff
@@ -28,7 +41,15 @@ ii. Once these are completed, reset MFA for [WOG account](reset-security-verific
 ?> In the service request form, select **Service Request** and **Request to reset Multi Factor Authentication (MFA)** as **Ticket Type** and **Service Requests**, respectively.
 
```

Comment:
what about If you are a public officer?

Reply:
is your question "what happens if a PO TechPass account gets locked?". I have made the question to be generic. Pls let me know if i have addressed your comment?
<summary>1. My TechPass account has got locked. How to unlock my TechPass account?</summary>

If you are a vendor, your TechPass account will be locked after five consecutive unsuccessful login attempts. Go to [reset password][reset-password] and follow the on-screen instructions.

If you are a public officer, [reset GSIB password][reset-password-gsib] according to WOG's password policies. If there are any issues, contact your Agency Facility Management (AFM)

Reply:
I have removed the PO reset GSIB pwd from this answer. Please clarify what is the ask here?

Reply:
ah... please ignore this comment.  I saw **If you are a vendor**:

and missed  **If you are a public officer**: